src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -101,7 +102,11 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                 var message = new HttpRequestMessage
                 {
                     Method = HttpMethod.Post,
-                    Content = _stream
+                    Content = _stream,
+                    Headers =
+                    {
+                        TransferEncoding = { new TransferCodingHeaderValue("chunked") }
```

Comment:
I think that there is 'TransferEncodingChunked' property in Headers available. Maybe we should use it instead?

Reply:
We didn't see that in the intellisense, possible 1.3 vs 2.0 issue?

Reply:
Or R# intellisense, everything compiles on my end:

```csharp
                var message = new HttpRequestMessage
                {
                    Method = HttpMethod.Post,
                    Content = _stream,
                    Headers =
                    {
                        TransferEncodingChunked = true
                    }
                };
```

Reply:
I'll make the change